src/OpenDiffix.Core/SQLNoiseLayers.fs

```diff
@@ -0,0 +1,42 @@
+module OpenDiffix.Core.SQLNoiseLayers
```

Comment:
Probably better to drop `SQL` from the module name, as we will likely also include functionality for the dynamic component of noise layers here.